plugins/plugin-chart-echarts/src/components/Echart.tsx

```diff
@@ -26,38 +26,54 @@ const Styles = styled.div<EchartsStylesProps>`
   width: ${({ width }) => width};
 `;
 
+const usePrevious = <T extends unknown>(value: T): T | undefined => {
```

Comment:
```suggestion
const usePrevious = <T>(value: T): T | undefined => {
```

I think this is good enough?